packages/fluentui/react-northstar-fela-renderer/src/felaInvokeKeyframesPlugin.ts

```diff
@@ -20,18 +20,15 @@ export const felaInvokeKeyframesPlugin = (styles: ICSSInJSStyle): ICSSInJSStyle
           styles[cssPropertyName] = callable(animationDefinition.keyframe)(animationDefinition.params || {});
         }
 
-        return {
-          ...acc,
-          [cssPropertyName]: styles[cssPropertyName],
-        };
+        acc[cssPropertyName] = styles[cssPropertyName];
```

Comment:
Typing the `acc` in the `reduce` call to `ICSSInJSStyle` should remove the compilation error

Reply:
ah yes, we're still on typescript 4.1.5 which does not have that issue I believe

Reply:
Could be related to #45149

Reply:
I would suggest linking that issue in the code to at least know why we are `any` casting here

Reply:
Please just add a comment about the complex union type in the code, otherwise it looks fine

Reply:
Added